### PR TITLE
Feat  add autocreate new xblock

### PR DESCRIPTION
**Pull request by connorhaugh**
This PR adds an automated script for generating the boilerplate code for adding a new xblock editor. As well as documentation for how to do so.

### CODE REVIEW

File: src/editors/containers/EditorContainer/index.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.pgn__modal-layer {
```
Comment (connorhaugh) [inline src/editors/containers/EditorContainer/index.scss:1]: We need to remove this again, right?

File: www/src/Gallery.jsx [hunk 2]
```diff
@@ -26,9 +26,10 @@ export const EditorGallery = () => {
           onChange={handleChange}
           value={blockType}
         >
-          <Form.Radio value="html">Text</Form.Radio>
-          <Form.Radio value="video">Video</Form.Radio>
-          <Form.Radio value="problem">Problem</Form.Radio>
+          { blockTypes.map((e) => (
+            <Form.Radio value={e}> {e} </Form.Radio>
+          ))}
+
```
Comment (rayzhou-bit) [inline www/src/Gallery.jsx:32]: blockTypes is not an array here and will trigger an error.
Something like the following code should work.
```
          { Object.values(blockTypes).map((e) => (
            <Form.Radio value={e}> {e} </Form.Radio>
          ))}
```

**Review by rayzhou-bit — CHANGES_REQUESTED** [review]
There's one bit of code that should be changed. Otherwise this looks really nice. Very simple and elegant solution to adding a new block.

**Review by rayzhou-bit — APPROVED** [review]
lgtm!

**Pull request merged**